src/pages/generate/hooks/useFurnitureHotspots.ts

```
@@ -0,0 +1,450 @@
+    return () => cancelAnimationFrame(raf);
+  }, [
+    enabled,
+    updateRenderMetrics,
+    state.containerSize.width,
+    state.containerSize.height,
+  ]);
+
+  useEffect(() => {
+    if (!state.imageMeta || !state.renderMetrics) {
+      dispatch({
+        type: 'PROJECTED_READY',
```

ESLint

[error] Replace `'PROJECTED_READY'` with `"PROJECTED_READY"`

(prettier/prettier)